src/libutil/regex-combinators.hh

```diff
@@ -10,22 +12,17 @@ namespace nix::regex {
 
 static inline std::string either(std::string_view a, std::string_view b)
 {
-    return std::string { a } + "|" + b;
+    return std::format("{}|{}", a, b);
```

Comment:
Not sure if all our C++ compilers support this yet?

Reply:
Looks like macos doesn't...

Reply:
Found a decent alternative.